src/de/dkfz/b080/co/aceseq/ACESeqWorkflow.java

```diff
@@ -76,20 +85,21 @@ public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged,
 
 
         Tuple2<TextFile, TextFile> breakpoints = ACESeqMethods.pscbsGaps(haplotypedSNPFile, correctedWindowFile.value0, annotationResult.getGenderFile());
-        Tuple2<TextFile, TextFile> mergedSvs = null;
+        Tuple2<SVFile, TextFile> mergedSvs = null;
 
-        if (runWithSv) {
-            mergedSvs = ACESeqMethods.mergeSv(breakpoints.value0, runWithSv); // true is passed
+        if (runWithSV) {
+            mergedSvs = ACESeqMethods.mergeSv(breakpoints.value0, runWithSV); // true is passed
+            if (mergedSvs == null) {
+                return allowMissingSVFile(context); // It might be allowed to exit here without an error
+            }
         } else if (runWithCrest) {
             mergedSvs = ACESeqMethods.mergeCrest(breakpoints.value0);
+            if (mergedSvs == null)
+                return false;  // Getting no merged SVs from the Crest step is always wrong. It is not all
         } else {
-            ACESeqMethods.mergeSv(breakpoints.value0, runWithSv); // false is passed
-            return true;
+            mergedSvs = ACESeqMethods.mergeSv(breakpoints.value0, runWithSV); // false is passed
```

Comment:
What do you mean by "false is passed"? Here nothing is passed out, and passed in is a tuple.

Reply:
rubWithSVs is false... Redundant so can be remived